rescript/parse_rdp.py

```diff
@@ -0,0 +1,89 @@
+# ----------------------------------------------------------------------------
```

Comment:
I recommend tossing this code in the `get_rdp_data.py` file for easier organization (rescript has a lot of different module files at the moment...)